model/metrics/system-metrics.yaml

```diff
@@ -340,3 +340,17 @@ groups:
     instrument: updowncounter
     unit: "By"
     attributes: []
+
+  - id: metric.system.linux.memory.slab.usage
```

Comment:
should it be `linux` or `unix`?

Reply:
As far as I am aware, it should be `linux`. Although, some other Unix-like kernels like Solaris uses the same memory management strategy, others use a different implementation or name. For example, FreeBSD uses UMA (Universal Memory Allocator) which is a distinct implementation optimized for the FreeBSD kernel. (OpenBSD uses the Pool Allocator)